cli/src/commands/parse.rs

```diff
@@ -21,7 +21,6 @@ const LOCKFILE_PARSERS: &[(&str, &dyn Fn(&Path) -> ParserResult)] = &[
     ("mvn", &parse::<Pom>),
     ("gradle", &parse::<GradleDeps>),
     ("nuget", &parse::<CSProj>),
-    ("auto", &get_packages_from_lockfile),
```

Comment:
I think you will need to update the `lockfile_types()` function to still return `"auto"` in its list so that `clap` will accept `-t auto`